teamB/jeonghun/app.js

```diff
@@ -0,0 +1,123 @@
+require('dotenv').config();
+const http = require ('http');
+const express = require('express');
+const cors = require('cors');
+const morgan = require('morgan');
+
+const {DataSource} = require('typeorm');
+
+const appDataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE
+});
+
+appDataSource.initialize()
+ .then(() => {
+  console.log('Data Source has been initialized!')  
+ })
+ .catch((error) => {
+  console.error('Error during Data source initialization', error);
+ });
+
+const app = express()
+
+const server = http.createServer(app)
+const PORT = process.env.PORT;
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan('dev'));
+
+app.get('/ping', function (req, res, next) {
+    res.status(201).json({message: 'pong'});
+});
```

Comment:
상태코드 201은 데이터가 생성 되었을 때 사용합니다. 단순 ok일 경우 200을 사용하시면 됩니다.

```suggestion
app.get('/ping', function (req, res, next) {
    res.status(200).json({message: 'pong'});
});
```